lib/active_shipping/errors.rb

```diff
@@ -2,6 +2,9 @@ module ActiveShipping
   class Error < ActiveUtils::ActiveUtilsError
   end
 
+  class RequestError < ActiveShipping::Error
```

Comment:
We could probably just change `ActiveShipping::Error` to inherit directly from `StandardError`, no?

What's the benefit of `ActiveUtils::ActiveUtilsError`?